py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -135,6 +135,9 @@ class RemoteConnection(object):
 
     Communicates with the server using the WebDriver wire protocol:
     https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol"""
+
+    timeout = socket._GLOBAL_DEFAULT_TIMEOUT
```

Comment:
@AutomatedTester, thank you for you comment. And actually I need to update this pull-request, because in our custom selenium package, we found, that it's not enough to support socket timeout, because it need to catch socket.error too. I will update